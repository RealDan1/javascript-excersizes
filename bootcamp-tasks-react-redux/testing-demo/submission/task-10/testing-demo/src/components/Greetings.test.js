import Greetings from './Greetings';
import React from 'react';
import renderer from 'react-test-renderer';

test('Greetings says Hello World', () => {
    const tree = renderer.create(<Greetings />).toJSON();
    expect(tree).toMatchSnapshot();
});
