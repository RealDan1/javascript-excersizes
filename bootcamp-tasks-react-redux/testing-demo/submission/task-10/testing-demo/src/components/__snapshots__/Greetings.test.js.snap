// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greetings says Hello World 1`] = `
<div>
  <p>
    Hello World
  </p>
</div>
`;
